### PR TITLE
G3 2024 v2 #14533

### PR DESCRIPTION
**Pull request by a22maxra**
Added a check to if there is a Github URL set for the course when trying to get the latest fetch.

### CODE REVIEW

**Review by a22moamy — CHANGES_REQUESTED** [review]
This fix creates the following error alert on my mac: 
Skärmavbild 2024-04-16 kl  10 29 22

And then crashes the website as can be seen below:
Skärmavbild 2024-04-16 kl  10 29 28

Please fix this error and request another review.

**Comment by a22maxra** [comment]
Added error some handling

**Review by a22moamy — CHANGES_REQUESTED** [review]
# Small fix required
When you add a github repo URL to a pre-existing course by clicking the "fetch github repo" button, it does not immediately respond to the new URL. After adding the URL, it does not update visually and looks like this: 
Skärmavbild 2024-04-16 kl  14 44 10

Currently, for the tooltip to update, the user has to refresh the page. **Please try to make it so that the user does not have to refresh the page after adding a github repo to a course through the "fetch github repo" button.** If this is not possible, please state why. 

"fetch github repo" button: 
Skärmavbild 2024-04-16 kl  14 51 17

Besides this small fix, the implementation works as intended. Since this branch is behind a merge that fixes new courses, I cannot fully guarantee that it works on new courses.

File: DuggaSys/sectioned.js [hunk 1]
```diff
@@ -3830,6 +3830,8 @@ function validateForm(formid) {
         localStorage.setItem('courseGitHubRepo', repoLink);
         $("#githubPopupWindow").css("display", "none");
         updateGithubRepo(repoLink, cid);
+        location.reload();
+        console.log("TESTAR, RELOADED DET NYSS???");
```
Comment (b22marah) [inline DuggaSys/sectioned.js:3834]: REMOVE

**Comment by b22marah** [comment]
Previous comments has been fixed, and both me and tester with these fixes thinks it looks good.

**Pull request merged**